pkg/recorder/recorder.go

```diff
@@ -103,6 +103,10 @@ func (r *Recorder) Run() *livekit.RecordingInfo {
 			})
 		}
 	case *livekit.StartRecordingRequest_Filepath:
+		if r.conf.FileOutput.StreamingUpload != nil {
```

Comment:
this would still be considered a FileResult, and should return info with duration like below